src/python/pants/backend/python/subsystems/pytest.py

```diff
@@ -44,6 +44,29 @@ def register_options(cls, register):
              help='Requirements string for the unittest2 library, which some python versions '
                   'may need.',
              removal_version="1.25.0.dev0", removal_hint="Use --pytest-plugins instead.")
+    register(
+      '--timeouts',
```

Comment:
Thoughts on renaming this? I've never liked that the namespace is different than the other `--timeout` options. Ideally, this would be something like `--timeout-enabled` or `--timeout-disabled`, imo.

I believe the main use case for users is to on the CLI pass `--no-timeouts` when they want to debug something. That is, usually people like it defaulting to True and only change it in specific one-off instances. If this is the case, then `--pytest-timeout-disabled` would probably be the best.

(Please feel free to disagree. It's totally possible we should stick with `--timeouts`.)

Reply:
I'm in full agreement with you here. The naming seemed weird to me, and I agree the main use case I can see for it is `--no-timeouts` when I'm dropping a trace somewhere in a test.